numpy/_core/code_generators/ufunc_docstrings.py

```diff
@@ -4616,7 +4616,71 @@ def add_newdoc(place, name, doc):
 
     """)
 
-add_newdoc('numpy.core.umath', '_replace',
+add_newdoc('numpy._core.umath', 'index',
+    """
+    Like `find`, but raises :exc:`ValueError` when the substring is not found.
+
+    Parameters
+    ----------
+    x1 : array_like, with ``bytes_`` or ``unicode_`` dtype
```

Comment:
Commenting to remind myself to update all of these to mention stringdtype as well. No need to fix this yourself.

Reply:
I just pushed a commit to this branch to fix these docstrings.